### PR TITLE
name generator

### PR DESCRIPTION
**Pull request by sebinside**
i'm very sorry

### CODE REVIEW

**Comment by sebinside** [comment]
(must be squashed, I hate IntellIJ handling commit mesages)

File: jplag/src/main/java/jplag/CLI.java [hunk 1]
```diff
@@ -20,7 +21,21 @@
  */
 public class CLI {
 
-    private static final String DESCRIPTION = "JPlag - Detecting Software Plagiarism";
+    private static final String[] DESCRIPTIONS = {
+            "Detecting Software Plagiarism",
+            "Archaeological Playground",
+            "Since 1994",
+            "Scientifically Published",
+            "Supported by SDQ",
```
Comment (tsaglam) [inline jplag/src/main/java/jplag/CLI.java:29]: I vote for "Maintained by"

File: jplag/src/main/java/jplag/CLI.java [hunk 2]
```diff
@@ -20,7 +21,21 @@
  */
 public class CLI {
 
-    private static final String DESCRIPTION = "JPlag - Detecting Software Plagiarism";
+    private static final String[] DESCRIPTIONS = {
+            "Detecting Software Plagiarism",
+            "Archaeological Playground",
```
Comment (tsaglam) [inline jplag/src/main/java/jplag/CLI.java:26]: How about "Software-Archaeological"

**Review by tsaglam — CHANGES_REQUESTED** [review]

File: jplag/src/main/java/jplag/CLI.java (hunk 2 above)
Comment (sebinside) [reply jplag/src/main/java/jplag/CLI.java:26]: i thought the playground thing would be less direct :D

**Review by tsaglam — APPROVED** [review]

**Pull request merged**